webapp/src/main/webapp/iaf/frankdoc/views/attribute-description.html

```diff
@@ -0,0 +1,20 @@
+<div>
+  <div ng-if="! attr.enum" ng-bind-html="attr.description" />
+  <div ng-if="attr.enum">
+    <div ng-bind-html="attr.description" />
+    <p>
+    <p>
+    <table class="attributeEnumValues">
+      <thead>
+        <tr>
+          <th>Value</th>
+          <th>Meaning</th>
+        </tr>
+      </thead>
+      <tr ng-repeat="enumValue in ::enumValuesOfAttribute(attr)">
```

Comment:
Je hebt het inderdaad goed begrepen. Aangezien deze functie nu niet meer veel doet, had ik voorgesteld direct `::enums[attr.enum]` te gebruiken, dat neemt minder text in beslag en doet het zelfde :)

Reply:
Gedaan.